glide.yaml

```diff
@@ -6,7 +6,6 @@ owners:
   email: rob@tigera.io
 import:
 - package: github.com/docopt/docopt-go
-- package: github.com/ghodss/yaml
```

Comment:
I think instead of doing this, glide lets you mask one as the other.

Not sure if that's preferred. Something like this:

```
- package: github.com/ghodss/yaml
   version: calico
   repo: github.com/projectcalico/go-yaml-wrapper
```